doc/tutorial/single_cell_model.rst

```diff
@@ -176,6 +176,9 @@ single spike should be generated at around the same time the stimulus we
 provided in step (3) gets activated (10ms).
 
 And, finally, we plot the membrane potential
+.. literalinclude:: ../../python/example/single_cell_model.py
+   :language: python
+   :lines: 6-7
 
```

Comment:
Question: What does the second include do here?

Reply:
import pandas and seaborn hehe

Reply:
So, when you preview the docs here:
https://arbor--2288.org.readthedocs.build/en/2288/tutorial/single_cell_model.html
It renders incorrectly, I assume you need an empty line before `literalinclude`